app/public/utils/filtering/filterUtils.js

```diff
@@ -52,3 +52,5 @@ export const filtersFromUrl = (url) => url.href.split('&').filter((item) => isFi
     type: filterType(item),
     search: filterSearch(item),
 }));
+
+export const buildFilterPhrase = (field, value, type = null) => `filter[${field}]${type ? `[${type}]` : ''}=${value}`;
```

Comment:
I propose to change name of third argument `type` to `operator`, it's rather more descriptive as here should be put stuff like 'like', 'in', 'gt', etc.

Reply:
Yes, that sounds better. I've changed that already.